moveit_planners/pilz_industrial_motion_planner/include/pilz_industrial_motion_planner/joint_limits_container.h

```diff
@@ -128,17 +128,33 @@ class JointLimitsContainer
    * @brief verify position limit of single joint
    * @param joint_name
    * @param joint_position
-   * @return
+   * @return true if within limits, false otherwise
+   */
+  bool verifyPositionLimit(const std::string& joint_name, const double& joint_position) const;
+
+  /**
+   * @brief verify velocity limit of single joint
+   * @param joint_name
+   * @param joint_velocity
+   * @return true if within limits, false otherwise
    */
   bool verifyVelocityLimit(const std::string& joint_name, const double& joint_velocity) const;
 
   /**
-   * @brief verify position limit of single joint
+   * @brief verify acceleration limit of single joint
    * @param joint_name
-   * @param joint_position
-   * @return
+   * @param joint_acceleration
+   * @return true if within limits, false otherwise
    */
-  bool verifyPositionLimit(const std::string& joint_name, const double& joint_position) const;
+  bool verifyAccelerationLimit(const std::string& joint_name, const double& joint_acceleration) const;
```

Comment:
Odd question, why pass a const-ref to a double? Doesn't that just add indirection by creating a pointer to a value of the same size as the pointer?